src/features/hotspots/details/HotspotDetailsHandle.tsx

```diff
@@ -1,28 +1,41 @@
-import { Hotspot } from '@helium/http'
-import React, { memo } from 'react'
+import React, { memo, useCallback } from 'react'
+import Chevron from '@assets/images/chevron-right.svg'
+import { StyleSheet } from 'react-native'
 import Box from '../../../components/Box'
-import CardHandle from '../../../components/CardHandle'
-import HotspotMoreMenuButton from './HotspotMoreMenuButton'
-import FollowButton from '../../../components/FollowButton'
+import TouchableOpacityBox from '../../../components/TouchableOpacityBox'
+import { useAppDispatch } from '../../../store/store'
+import hotspotsSlice from '../../../store/hotspots/hotspotsSlice'
+
+const HotspotDetailsHandle = () => {
+  const dispatch = useAppDispatch()
+
+  const prev = useCallback(() => {
+    dispatch(hotspotsSlice.actions.selectPrevHotspot())
+  }, [dispatch])
+
+  const next = useCallback(() => {
+    dispatch(hotspotsSlice.actions.selectNextHotspot())
+  }, [dispatch])
 
-type Props = { hotspot: Hotspot }
-const HotspotDetailsHandle = ({ hotspot }: Props) => {
   return (
     <Box
       flexDirection="row"
       flex={1}
       justifyContent="space-between"
       alignItems="center"
     >
-      <FollowButton
-        paddingHorizontal="l"
-        paddingVertical="m"
-        address={hotspot.address}
-      />
-      <CardHandle />
-      <HotspotMoreMenuButton hotspot={hotspot} />
+      <TouchableOpacityBox onPress={prev} padding="m" style={styles.prevButton}>
+        <Chevron color="#C2C5E4" height={20} width={20} />
+      </TouchableOpacityBox>
+      <TouchableOpacityBox onPress={next} padding="m">
```

Comment:
Should we remove the arrows if you haven't selected on your own or followed hotspots?